### ISSUE TITLE
Problem Party with multiple bots (3)

### ISSUE THREAD

**Issue by Bunka** [kolton/d2bot-with-kolbot#1171]
Hi everyone,
Im getting issues when running 3 bots, everytimes, the 3rd profile joining the game won't accept invite frome the 2 others, no matter the order they join with the JoinDelay (line 35 from D2BotFollow), no matter if i put 1 or 2 with the Config.PublicMode in the different profiles.
Any tips ?
Thx

**Comment by Smas87** [comment]
i have the same issue as well need help =)

**Comment by Bunka** [comment]
Hey guys !
Thx for your notes.
@drogga i think i've tried pretty much every logical combination, but i might missing something somewhere...
@fabio1999ita do you mean that the leader have to enter as 3rd position in the game ?

**Comment by fabio1999ita** [comment]
No, you don't touch file of follower, you need set leader in charconfig of follwer and in leaderconfig of char you need set send invite and accept, maybe post all 3 charconfig

**Comment by aim2kill** [comment]
Set leader to inv only and follower to accept only

**Comment by Bunka** [comment]
### Leader lines 128 to 130 (Paladin profile) : 
	
Config.Leader = ""; // Leader's ingame character name. Leave blank to try auto-detection (works in AutoBaal, Wakka, MFHelper)
	Config.QuitList = ["Bunka1", "Bunka2"]; // List of character names to quit with. Example: Config.QuitList = ["MySorc", "MyDin"];
	Config.QuitListMode = 1; // 0 = use character names; 1 = use profile names (all profiles must run on the same computer).

### Leader lines 407 to 421 (Paladin profile) : 

	// If LocalChat is enabled, chat can be sent via 'sendCopyData' instead of BNET
	// To allow 'say' to use BNET, use 'say("msg", true)', the 2nd parameter will force BNET
	// LocalChat messages will only be visible on clients running on the same PC
	Config.LocalChat.Enabled = false; // enable the LocalChat system
	Config.LocalChat.Toggle = false; // optional, set to KEY value to toggle through modes 0, 1, 2
	Config.LocalChat.Mode = 0; // 0 = disabled, 1 = chat from 'say' (recommended), 2 = all chat (for manual play)
	// If Config.Leader is set, the bot will only accept invites from leader. If Config.PublicMode is not 0, Baal and Diablo script will open Town Portals.
	Config.PublicMode = 3; // 1 = invite and accept, 2 = accept only, 3 = invite only, 0 = disable
	// Party message settings. Each setting represents an array of messages that will be randomly chosen.
	// $name, $level, $class and $killer are replaced by the player's name, level, class and killer
	Config.Greetings = []; // Example: ["Hello, $name (level $level $class)"]
	Config.DeathMessages = []; // Example: ["Watch out for that $killer, $name!"]
	Config.Congratulations = []; // Example: ["Congrats on level $level, $name!"]
	Config.ShitList = false; // Blacklist hostile players so they don't get invited to party.
	Config.UnpartyShitlisted = false; // Leave party if someone invited a blacklisted player.

### Follower 1 lines 128 to 130 (Sorc profile) : 

	Config.Leader = "Bunka4"; // Leader's ingame character name. Leave blank to try auto-detection (works in AutoBaal, Wakka, MFHelper)
	Config.QuitList = ["Bunka2", "Bunka4"]; // List of character names to quit with. Example: Config.QuitList = ["MySorc", "MyDin"];
	Config.QuitListMode = 1; // 0 = use character names; 1 = use profile names (all profiles must run on the same computer).

### Follower 1 lines 407 to 421 (Sorc profile): 

	// If LocalChat is enabled, chat can be sent via 'sendCopyData' instead of BNET
	// To allow 'say' to use BNET, use 'say("msg", true)', the 2nd parameter will force BNET
	// LocalChat messages will only be visible on clients running on the same PC
	Config.LocalChat.Enabled = false; // enable the LocalChat system
	Config.LocalChat.Toggle = false; // optional, set to KEY value to toggle through modes 0, 1, 2
	Config.LocalChat.Mode = 0; // 0 = disabled, 1 = chat from 'say' (recommended), 2 = all chat (for manual play)
	// If Config.Leader is set, the bot will only accept invites from leader. If Config.PublicMode is not 0, Baal and Diablo script will open Town Portals.
	Config.PublicMode = 2; // 1 = invite and accept, 2 = accept only, 3 = invite only, 0 = disable
	// Party message settings. Each setting represents an array of messages that will be randomly chosen.
	// $name, $level, $class and $killer are replaced by the player's name, level, class and killer
	Config.Greetings = []; // Example: ["Hello, $name (level $level $class)"]
	Config.DeathMessages = []; // Example: ["Watch out for that $killer, $name!"]
	Config.Congratulations = []; // Example: ["Congrats on level $level, $name!"]
	Config.ShitList = false; // Blacklist hostile players so they don't get invited to party.
	Config.UnpartyShitlisted = false; // Leave party if someone invited a blacklisted player.

### Follower 2 lines 128 to 130 (Sorc profile): 

	Config.Leader = "Bunka4"; // Leader's ingame character name. Leave blank to try auto-detection (works in AutoBaal, Wakka, MFHelper)
	Config.QuitList = ["Bunka1", "Bunka4"]; // List of character names to quit with. Example: Config.QuitList = ["MySorc", "MyDin"];
	Config.QuitListMode = 1; // 0 = use character names; 1 = use profile names (all profiles must run on the same computer).

### Follower 2 lines 407 to 421 (Sorc profile):

	// If LocalChat is enabled, chat can be sent via 'sendCopyData' instead of BNET
	// To allow 'say' to use BNET, use 'say("msg", true)', the 2nd parameter will force BNET
	// LocalChat messages will only be visible on clients running on the same PC
	Config.LocalChat.Enabled = false; // enable the LocalChat system
	Config.LocalChat.Toggle = false; // optional, set to KEY value to toggle through modes 0, 1, 2
	Config.LocalChat.Mode = 0; // 0 = disabled, 1 = chat from 'say' (recommended), 2 = all chat (for manual play)
	// If Config.Leader is set, the bot will only accept invites from leader. If Config.PublicMode is not 0, Baal and Diablo script will open Town Portals.
	Config.PublicMode = 2; // 1 = invite and accept, 2 = accept only, 3 = invite only, 0 = disable
	// Party message settings. Each setting represents an array of messages that will be randomly chosen.
	// $name, $level, $class and $killer are replaced by the player's name, level, class and killer
	Config.Greetings = []; // Example: ["Hello, $name (level $level $class)"]
	Config.DeathMessages = []; // Example: ["Watch out for that $killer, $name!"]
	Config.Congratulations = []; // Example: ["Congrats on level $level, $name!"]
	Config.ShitList = false; // Blacklist hostile players so they don't get invited to party.
	Config.UnpartyShitlisted = false; // Leave party if someone invited a blacklisted player.

### D2BotFollow.dbj lines 31 to 33 : 

var JoinSettings = {
	"Bunka4": ["all"]
};


@fabio1999ita 
@aim2kill 

Still doesn't work like this, i've tried to delet/reinstall D2bot to see if it was changing anything but it seems not to.

**Comment by fabio1999ita** [comment]
is normal that not work, you write profile name and for work you need write charname
Config.Leader = "charnameingame"; //not profile name

**Comment by smallsi1028** [comment]
Config.Leader = "charnameingame"; //not profile name

Check that the lowercase and uppercase of charnameingame is correct.
For example:
  Your char name in game is "HolyPaladin".
  So you need to type this :
    Config.Leader = "HolyPaladin";
  If you type :
    Config.Leader = "holypaladin";  
  it is wrong.
